modelling/src/main/java/org/axonframework/modelling/configuration/ModellingConfigurer.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2010-2025. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.modelling.configuration;
+
+import jakarta.annotation.Nonnull;
+import org.axonframework.configuration.ApplicationConfigurer;
+import org.axonframework.configuration.AxonApplication;
+import org.axonframework.configuration.Component;
+import org.axonframework.configuration.ComponentDecorator;
+import org.axonframework.configuration.ConfigurationEnhancer;
+import org.axonframework.configuration.DelegatingConfigurer;
+import org.axonframework.configuration.MessagingConfigurer;
+import org.axonframework.configuration.Module;
+import org.axonframework.configuration.ModuleBuilder;
+import org.axonframework.configuration.NewConfigurer;
+
+import java.util.function.Consumer;
+
+/**
+ * The modelling {@link NewConfigurer} of Axon Framework's configuration API, providing registration methods to, for
+ * example, register a {@link StatefulCommandHandlingModule}.
+ * <p>
+ * This configurer does not set any defaults other than the defaults granted by the {@link MessagingConfigurer} it
+ * wraps.
+ *
+ * @author Steven van Beelen
+ * @since 5.0.0
+ */
+public class ModellingConfigurer
+        extends DelegatingConfigurer<ModellingConfigurer>
+        implements ApplicationConfigurer<ModellingConfigurer> {
+
+    /**
+     * Build a default {@code ModellingConfigurer} instance with several modelling defaults.
```

Comment:
There are no defaults right? ;) Like in line 37.
```suggestion
     * This configurer does not set any defaults other than the defaults granted by the {@link MessagingConfigurer} it 
      * wraps.
```